<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vision System</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <activeon>:/icone/imagens/oie_transparent.png</activeon>
   </iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="janela">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Comandos</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Estrategia</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QComboBox" name="comboBox"/>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Atacante</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QComboBox" name="combo_atacante"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Zagueiro</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QComboBox" name="combo_zagueiro"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Goleiro</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="2">
        <widget class="QComboBox" name="combo_goleiro"/>
       </item>
       <item row="5" column="0" colspan="3">
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>280</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="Tempo">
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Tempo:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="milesegundos">
        <property name="maximumSize">
         <size>
          <width>15</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Tempoms">
        <property name="text">
         <string>ms</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
   <zorder>groupBox</zorder>
   <zorder>janela</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>775</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Classificadores">
    <property name="title">
     <string>&amp;Opcoes</string>
    </property>
    <addaction name="actionAbrir_Settings"/>
    <addaction name="actionKmeans"/>
   </widget>
   <widget class="QMenu" name="menuTime">
    <property name="title">
     <string>&amp;Time</string>
    </property>
    <addaction name="select_azul"/>
    <addaction name="select_amarelo"/>
   </widget>
   <addaction name="menu_Classificadores"/>
   <addaction name="menuTime"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionKmeans">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Constantes</string>
   </property>
  </action>
  <action name="actionFind_Contours">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Find Contours</string>
   </property>
  </action>
  <action name="actionComandos">
   <property name="text">
    <string>Comandos</string>
   </property>
  </action>
  <action name="actionAzul">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Azul</string>
   </property>
  </action>
  <action name="actionAmarelo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Amarelo</string>
   </property>
  </action>
  <action name="select_azul">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Azul</string>
   </property>
  </action>
  <action name="select_amarelo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Amarelo</string>
   </property>
  </action>
  <action name="actionAbrir_Settings">
   <property name="text">
    <string>Abrir Settings</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
